lib/tasks/update_used_aliquots_with_primary_aliquot_tags.rake

```diff
@@ -3,9 +3,13 @@
 namespace :used_aliquots do
   task update_tags: :environment do
     libraries = Pacbio::Library.all.reject { |library| library.used_aliquots.first.tag == library.tag }
+
     libraries.each do |library|
       library.used_aliquots.each do |used_aliquot|
         used_aliquot.update!(tag: library.tag)
+        library.sequencing_runs.each do |run|
```

Comment:
Looks like Messages publish can take a list of objects. So I think you can do `Messages.publish(library.sequencing_runs, Pipelines.pacbio.message)`. Saves having to iterate over the runs twice.